client/controllers/smartEvents/smartEvent.coffee

```diff
@@ -11,6 +12,10 @@ Template.smartEvent.onCreated ->
   @filterQuery = new ReactiveVar({})
   @selectedIncidentTypes = new ReactiveVar([])
 
+  @hasNoIncidents = =>
+    filterQuery = @filterQuery.get()
+    _.isEmpty(filterQuery) and not EventIncidents.find(filterQuery).count()
```

Comment:
Oh, right. I guess we only need to check if the empty query has a count.
This brings up an issue—if the query is empty (the user creates a smart event with only a title) doesn't the smart event page try to subscribe to every incident?